src/sentry/relay/projectconfig_cache/redis.py

```diff
@@ -1,8 +1,28 @@
+import logging
+import random
+
+import zstandard
+
+from sentry import options
 from sentry.relay.projectconfig_cache.base import ProjectConfigCache
 from sentry.utils import json, metrics, redis
 from sentry.utils.redis import validate_dynamic_cluster
 
 REDIS_CACHE_TIMEOUT = 3600  # 1 hr
+COMPRESSION_OPTION_DSNS = "relay.project-config-cache-compress"
+COMPRESSION_OPTION_SAMPLE_RATE = "relay.project-config-cache-compress-sample-rate"
+COMPRESSION_LEVEL = 9  # TODO: is this too much for Relay decompression?
```

Comment:
did you experiment with the value? how much do we gain from this vs the default? I think `-9` can really burn a ton of CPU *during compression* (it should have little impact during decompression)

Reply:
Nope, I just went to the highest value, hoping that even that is fast enough with zstd. Changed it to `3` now, the default value, it's the safer bet.